packages/rule-tester/tests/RuleTester.test.ts

```diff
@@ -819,6 +819,37 @@ describe('RuleTester', () => {
         expect(mockedDescribeSkip.mock.calls).toHaveLength(0);
         // expect(mockedIt.mock.lastCall).toMatchInlineSnapshot(`undefined`);
       });
+
+      it('does not call describe with valid if no valid tests are provided', () => {
+        const ruleTester = new RuleTester();
+
+        ruleTester.run('my-rule', NOOP_RULE, {
+          valid: [],
+          invalid: [
+            {
+              code: 'invalid',
+              errors: [{ messageId: 'error' }],
+            },
+          ],
+        });
+
+        expect(mockedDescribe.mock.calls).toHaveLength(2);
+      });
+
+      it('does not call describe with invalid if no invalid tests are provided', () => {
+        const ruleTester = new RuleTester();
+
+        ruleTester.run('my-rule', NOOP_RULE, {
+          valid: [
+            {
+              code: 'valid',
+            },
+          ],
+          invalid: [],
+        });
+
+        expect(mockedDescribe.mock.calls).toHaveLength(2);
```

Comment:
These two tests have the same assertions. I'd think it good to verify that they're called with the right portions. Maybe `toHaveBeenCalledWith` or `toMatchInlineSnapshot` or something like that. Thoughts?

Reply:
Thanks! Did it with `toMatchInlineSnapshot`.